app/grandchallenge/evaluation/admin.py

```diff
@@ -44,13 +55,65 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.fields["inputs"].queryset = ComponentInterface.objects.exclude(
+            slug__in=NON_EVALUATION_INTERFACES
+        )
```

Comment:
Somehow this does not work. I still get the full list of CIs in the admin. Not sure what I'm doing wrong. 
Also, `NON_EVALUATION_INTERFACES` == `NON_ALGORITHM_INTERFACES`, but it seemed semantically wrong to use the latter here. We might also want to differentiate the two in the future.

Reply:
I think that the data from the autocomplete comes from how it is defined on the related model https://docs.djangoproject.com/en/5.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.autocomplete_fields

I think the only thing you can do here is to make sure that it is cleaned and rejected if selected.

Reply:
Ah that makes sense. Thanks! I'll add check in the clean method.